docs/analyze_dss.puml

```diff
@@ -29,12 +32,7 @@ A form with all the
 information needed
 by the inspector.
 end note
-F <<-- B: return Response
-F -> F: GET /form
-note right
-Open the page where the
-form will be displayed
-end note
+F <<-- B: return JSON
```

Comment:
I believe at some point in this process, a draft is saved?

Reply:
It's showcased in the workflow diagram. Once the initial form is returned, the user can sends back a form using `POST /forms` which creates a draft by default.